tests/output_filters/virustotal/lookup_domains_test.py

```diff
@@ -13,3 +13,57 @@ def test_no_domains(self):
         ]
 
         self.run_test(LookupDomainsFilter, input_blobs=input_blobs, expected_output_blobs=input_blobs)
+
+    def test_benign_domains(self):
+        input_blobs = [
+            {'osxcollector_domains': ['good.example.com'], 'dingo': 'bingo', 'apple': [3, 14]},
+            {'osxcollector_domains': ['good.example.co.uk'], 'bingo': 'bongo', 'orange': 'banana'}
+        ]
+
+        self.run_test(LookupDomainsFilter, input_blobs=input_blobs, expected_output_blobs=input_blobs)
+
+    def test_suspicious_domains(self):
+        input_blobs = [
+            {'osxcollector_domains': ['evil.example.com'], 'dingo': 'bingo', 'apple': [3, 14]},
+            {'osxcollector_domains': ['evil.example.co.uk'], 'bingo': 'bongo', 'orange': 'banana'}
+        ]
+        output_blobs = [
```

Comment:
Yes, but wouldn't we like the test to be more exhaustive then just checking whether the key was added? The key value is also something that we would like to test here.